makefiles/cargo-targets.inc.mk

```diff
@@ -95,7 +95,7 @@ $(APPLICATION_RUST_MODULE).module: $(CARGO_LIB) FORCE
 # This prevents cargo inside docker from creating them with root permissions
 # (should they not exist), and also from re-building everything every time
 # because the .cargo inside is as ephemeral as the build container.
-$(shell mkdir -p ~/.cargo/git ~/.cargo/registry)
+$(shell mkdir -p $(HOME)/.cargo/git $(HOME)/.cargo/registry)
```

Comment:
```suggestion
$(shell mkdir -p $HOME/.cargo/git $HOME/.cargo/registry)
```

Reply:
I don't think this will work, as `$HOME` would be the same as `$(H)OME`, but there is no variable `H`.

Reply:
But I think it should assign this to a variable...

Reply:
But the general approach is not super sane. https://github.com/RIOT-OS/RIOT/pull/21465 probably saner.